src/wallet/rpcwallet.cpp

```diff
@@ -2476,11 +2476,16 @@ UniValue fundrawtransaction(const JSONRPCRequest& request)
                             "1. \"hexstring\"           (string, required) The hex string of the raw transaction\n"
                             "2. options               (object, optional)\n"
                             "   {\n"
-                            "     \"changeAddress\"     (string, optional, default pool address) The bitcoin address to receive the change\n"
-                            "     \"changePosition\"    (numeric, optional, default random) The index of the change output\n"
-                            "     \"includeWatching\"   (boolean, optional, default false) Also select inputs which are watch only\n"
-                            "     \"lockUnspents\"      (boolean, optional, default false) Lock selected unspent outputs\n"
-                            "     \"feeRate\"           (numeric, optional, default not set: makes wallet determine the fee) Set a specific feerate (" + CURRENCY_UNIT + " per KB)\n"
+                            "     \"changeAddress\"         (string, optional, default pool address) The bitcoin address to receive the change\n"
+                            "     \"changePosition\"        (numeric, optional, default random) The index of the change output\n"
+                            "     \"includeWatching\"       (boolean, optional, default false) Also select inputs which are watch only\n"
+                            "     \"lockUnspents\"          (boolean, optional, default false) Lock selected unspent outputs\n"
+                            "     \"feeRate\"               (numeric, optional, default not set: makes wallet determine the fee) Set a specific feerate (" + CURRENCY_UNIT + " per KB)\n"
+                            "     \"subtractFeeFromAmount\" (array, optional) A json array with addresses.\n"
```

Comment:
Maybe mention after `This will not modify existing inputs, and will add one change output to the outputs` above that no existing outputs will be modified either unless subtractFeeFromAmount is specified.